app/templates/home/content-column.html

```diff
@@ -92,7 +92,7 @@ <h1 id="how">How do I use UAVCAN?</h1>
   First, read the <a href="/guide" target="_blank">Guide</a>. To learn the technical details, read the
   <a href="/specification" target="_blank">Specification</a>. Familiarize yourself with the
   <a href="https://github.com/UAVCAN/public_regulated_data_types">regulated DSDL repository</a>.
-  If you have questions or feedback, bring them to the <a href="https://forum.uavcan.org">forum</a>.
+  If you have questions or feedback, you can check out the <a href="https://forum.uavcan.org/t/the-uavcan-v1-faq/1394">FAQ</a> or bring them to the <a href="https://forum.uavcan.org">forum</a>!.
```

Comment:
Wrapping the line and fixing the typo!.

```suggestion
  If you have questions or feedback, you can check out the
  <a href="https://forum.uavcan.org/t/the-uavcan-v1-faq/1394">FAQ</a>
  or bring them to the <a href="https://forum.uavcan.org">forum</a>.
```